app/database/group_group_store_transitions.go

```diff
@@ -277,6 +277,10 @@ func (s *GroupGroupStore) Transition(action GroupGroupTransitionAction,
 		}
 		var oldActions []idWithAction
 
+		// Here we get current states for each childGroupID:
+		// the current state can be one of
+		// ("", "invitation_created", "join_request_created", "added_directly", "added_directly,leave_request_created")
+		// where "added_directly" means that childGroupID is a member of the parentGroupID
```

Comment:
But if "added_directly" does not necessarily mean "added_directly", why using this name ? 😄
In my head at least, it creates confusion. It just means "is_member" just using such a name.

Reply:
fixed